DOC-6284: Archive Service 1.1.1 and 1.0.6 (#2068)
corda/corda-docs-portal: content/en/tools/archiving-service/archiving-release-notes.md

## Corda Enterprise 4.11 and above

### Archive Service 1.1.1

Archive Service 1.1.1 is a patch release focused on resolving issues.

#### Fixed Issues

> * When creating a backup table, the Archive Service was creating a backup table index name greater than 30 characters. This caused problems with Oracle 11. This has now been resolved.

Reviewer: ```suggestion
* When creating a backup table, the Archive Service created a backup table index name greater than 30 characters. This caused problems with Oracle 11. 
```

> * When used with a Corda 4.11 node, it was possible for the Archive Service `create-snapshot` command to fail. This issue occurred when the node database was one of SQL Server, Oracle, or PostgreSQL, and the Archive Service was configured to use a backup schema. This issue has been resolved.

Reviewer: ```suggestion
* When used with a Corda 4.11 node, it was possible for the Archive Service `create-snapshot` command to fail. This issue occurred when the node database was one of SQL Server, Oracle, or PostgreSQL, and the Archive Service was configured to use a backup schema.
```

> * When done immediately after a `delete-vault` operation, the `import-snapshot` would fail to import transactions in an Archive Service regression. This issue has been resolved.

Reviewer: ```suggestion
* When done immediately after a `delete-vault` operation, the `import-snapshot` failed to import transactions in an Archive Service regression. 
```

## Corda Enterprise 4.10 and below

### Archive Service 1.0.6

Archive Service 1.0.6 is a patch release focused on resolving issues.

#### Fixed Issues

> * When the node database was Oracle and the Archive Service was configured to use a backup schema, it was possible for the Archive Service `create-snapshot` command to fail. This issue has been resolved.

Reviewer: ```suggestion
* When the node database was Oracle and the Archive Service was configured to use a backup schema, it was possible for the Archive Service `create-snapshot` command to fail.

> * When creating a backup table, the Archive Service was creating a backup table index name greater than 30 characters. This caused problems with Oracle 11. This has now been resolved.

Reviewer: as above

Author: updated
